### ISSUE TITLE
Geth on Windows - major issues - Did you even test?

### ISSUE THREAD

**Issue by dnaleor** [ethereum/ethereum-org#113]
Hi devs!

Did you test the client?
There are 2 major issues when using the client

1) when you make an error in a command, the client crashes and any further (valid) commands are not working. You need to restart the client

2) when copying text, it often "keeps rolling on the screen" and it just doesn't work. 

These are major issues I found out when i tried to send transactions this morning. I was there when the first transactions were send, but I couldn't send mine. It took me about 2 hours to fix it and I still had unexpected errors.

I am starting to think you released Frontier in this way so you could limit the amount of ETH send in the first hours, so the "insiders/devs/nerds" could dump first. Seriously guys, you should at least have a working command line client on ALL major platforms. This windows client was clearly NOT tested.

**Comment by obscuren** [comment]
We've tested our client thoroughly. We wouldn't feel comfortable releasing if we hadn't done so. It would be stupid to think we would release a consensus platform without the necessary testing. 

I'm sad to hear you had some issues but you must understand that Frontier is indeed for the tech savvy, it's stated clearly just about anywhere. If you could create reproducible test case we will look in to it.

**Comment by dnaleor** [comment]
I am tech savvy. Not on the level of a "dev" but I can code some, and know how to handle command line and stuff. I installed the client using chocolatey, generated the genesis block with the python script, etc. So I'm not a noob.

Reproducible ? I already told you how to crash it:

1) make an error in a command and try a new command afterwards that is valid. It won't work
2) try copying some command into the command line window. You'll see the rolling text on your screen. 

Anyway, I moved all my ether now, so I don't care anymore.
Good luck with this project, because you'll need it. Still a lot of work to do. 

I must say, it is a genius plan to hype Ethereum and make it as difficult as possible for new people to  start using it so there will only be a limited amount of Ether on the market while still "looking fair". Enjoy the p&d!
